### ISSUE TITLE
Form submit on Enter key press not working, if only one input field in form

### ISSUE THREAD

**Issue by mkrecek234** [atk4/ui#1789]
A Form with only one input field won't submit on enter key press. It will if there are two input fields or more...

Code to replicate:
```
$form = \Atk4\Ui\Form::addTo($app);
$form->addControl('id');
// If you uncomment the next line, it will work
//$form->addControl('id2');
```
Console warning on atkjs-ui-min.js 1:1102

**Comment by mvorisek** [comment]
can be reproduced on `demos/form/form.php` demo (1st form has one field)

currently, the submit even is triggered twice somehow:

![image](https://user-images.githubusercontent.com/2228672/168533016-4fc2251a-8b1e-43a3-be67-bb4ffe4af781.png)

**Comment by mvorisek** [comment]
@ibelar do you know how to fix this easily?

**Comment by mvorisek** [comment]
can be also reproduced on `/demos/interactive/modal.php` demo

click `Multi Step Modal` and then (when input in modal is focused) press Enter key, reload flashes very quickly, but nothing happen

when submit thru `Save`, the following is shown:

![image](https://user-images.githubusercontent.com/2228672/185766619-d70af062-f010-4ade-b960-f3d63c984b7f.png)

I would expect the same result after Enter key is pressed

**Comment by AdityaMathur25** [comment]
hi i want to work on it please let me know if i can contribute to it ?

**Comment by mvorisek** [comment]
PR is welcomed

**Comment by AdityaMathur25** [comment]
can you assing me this

**Comment by mvorisek** [comment]
fixed in Fomantic-UI 2.9.0 (merged into develop already)
